### ISSUE TITLE
Tugboat network issues 7/18 - 7/19/2024

### ISSUE THREAD

**Issue by edmund-dunn** [department-of-veterans-affairs/va.gov-cms#18695]
## User Story or Problem Statement
Tugboat is having issues with its agent staying up. 

## Description or Additional Context 
Tugboat started having issues yesterday, 7/18. Nick tried started trouble shooting. Issues persisted 7/19.

## Acceptance Criteria
- [ ] Get tugboat up reliably.
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

**Comment by nfpappas-oddball** [comment]
Problem statement: Tugboat's agent keeps crashing which i causing previews to freeze. Restarting tugboat seemed to fix the issues for a few hours. however it did re-occur 

Discovery: we discovered that the root volume , where logs are stored (/opt/tugboat/logs), was full. This was a result of turning on debug logging. This caused certain logs to reach multiple gigbytes large.

Solution: We grew the root volume from 80GB to 250. We compressed the old logs and then removed some. We also disabled debug logging.  We restarted the server and everything appeared operational again.
